modules/custom/az_migration/config/schema/az_migration.schema.yml

```
@@ -0,0 +1,22 @@
+az_migration.settings:
+  type: config_object
+  label: 'AZ Migration Settings'
+  mapping:
+    migrate_d7_protocol:
+      type: string
+      label: 'Protocol'
+    migrate_d7_filebasepath:
+      type: string
+      label: 'URL minus schema, no trailing slash.'
+    migrate_d7_public_path:
+      type: string
+      label: 'path to public source files, no trailing or leading slash.'
+    migrate_d7_private_path:
+      type: string
+      label: 'path to private source files, no trailing or leading slash.'
+    migrate_d7_temporary_path:
+      type: string
+      label: 'tmp filepath.'
+    az_migrate_allow_archived_paragraphs:
```

Technically this should not exist yet, but if it did, should probably change the name to not have a prefix
```suggestion
    allow_archived_paragraphs:
```

---

Was going to suggest this change 👍🏻